### PR TITLE
Updated README documentation and examples

### PR DESCRIPTION
**Pull request by eric-zhu-uw**
Updated readme with documentation and examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,122 +1,146 @@
...
 
-## General Workflow
-1. quarantine identifies a test is flaky and decides it should be added to the list of quarantined tests
-   &downarrow;
-2. A Jira ticket is automatically created to fix the flaky test
-   &downarrow;
-3. All subsequent builds will pass the quarantined flaky test, regardless if it actually passes or fails
-   &downarrow;
-4. The flaky test has been fixed and the Jira ticket is closed resulting in the test automatically being removed from the list of quarantined tests
+Quarantine provide a run-time solution to diagnosing and disabling flaky tests and automates the workflow around test suite maintainence.
```
Comment (kevinmsun) [inline README.md]: *provides. maintenance is also typo'd

File: README.md [hunk 2]
```diff
@@ -1,122 +1,146 @@
...
-1. Create tables in your database required for the gem the upload and pull quarantined test info
-```
-bundle exec quarantine_dynamodb -h                      # see all options
+The quarantine gem supports CI pipelines:
+- [Buildkite](https://buildkite.com/docs/tutorials/getting-started)
+
+If you are interested in using quarantine but it does not support your CI or testing framework, feel free to reach out or create an issue and we can try to make it happen.
+
+## Purpose
+Flaky tests impact engineering velocity, reduce faith in test reliablity and give a false representation of code coverage. Manging flaky tests is a clunky process that involves constant build monitorization, difficult diagnosis and manual ticket creation. As a result, here at Flexport, we have created a Gem to automate the entire process to help improve the workflow and keep our massive test suites in prestine condition.
```
Comment (kevinmsun) [inline README.md]: manging -> managing?

prestine -> pristine

File: README.md [hunk 3]
```diff
@@ -1,122 +1,146 @@
...
 
-2. In your `spec_helper.rb` include the quarantine and rspec-retry gem
+And then execute:
 ```
-require "quarantine"
-require "rspec-retry"
+bundle install
 ```
-3. Configure rspec-retry to retry failed tests
+(might wanna move this to a later step)
```
Comment (kevinmsun) [inline README.md]: is the intention to publish this openly? if so, wanna -> want to

File: README.md [hunk 4]
```diff
@@ -1,122 +1,146 @@
...
+  --aws_region us-west-1              # with "quarantine_list" and "master_failed_tests"
+                                      # table names
+```
 
-## How to Set-up Development Locally
+You are all set to start quarantining tests!
 
-1. Setup dynamodb table names in AWS and add configurations
+## Try Quarantining Tests Locally
+Add a test that will flake consistently
```
Comment (kevinmsun) [inline README.md]: "flake consistently" is a bit of an oxymoron

Comment (eric-zhu-uw) [reply README.md]: I agree they contradict. I removed `consistently`.

File: README.md (hunk 3 above)
Comment (eric-zhu-uw) [reply README.md]: Good catch! It was just a reminder to add the db table creation to a different step.

**Pull request merged**